contracts/nft/src/execute/validate.rs

```diff
@@ -0,0 +1,24 @@
+use cosmwasm_std::{Coin, Deps};
+
+use crate::core::error::ContractError;
+use crate::{core::msg::ExecuteMsg, util::validate::Validate};
+
+impl Validate for ExecuteMsg {
+    fn validate(&self, _deps: Deps) -> Result<(), ContractError> {
+        Ok(())
```

Comment:
Is there any type of validation that we can add, or are we letting the created metadata messages do the validation?